src/parser/formatting-evaluator-module.ts

```diff
@@ -84,6 +72,35 @@ export class FormattingEvaluatorModule implements Module {
     return result;
   }
 
+  private _calculateFormattingTotal(
+    formatting: ReturnType<typeof this._getFormattingScore>["formatting"],
+    multiplierFactor: Multiplier
+  ): Decimal {
+    if (!formatting) return new Decimal(0);
+
+    return Object.values(formatting)
+      .reduce((acc, curr) => {
+        let sum = new Decimal(0);
+
+        for (const symbol of Object.keys(curr.symbols)) {
+          const count = new Decimal(curr.symbols[symbol].count);
+          const symbolMultiplier = new Decimal(curr.symbols[symbol].multiplier);
+          const formattingElementScore = new Decimal(curr.score);
+          const exponent = this._wordCountExponent;
+
+          sum = sum.add(
+            count
+              .pow(exponent) // (count^exponent)
+              .mul(symbolMultiplier) // symbol multiplier
+              .mul(formattingElementScore) // comment type multiplier
+              .mul(multiplierFactor.multiplier) // formatting element score
+          );
+        }
+        return acc.add(sum);
+      }, new Decimal(0))
+      .toDecimalPlaces(2);
```

Comment:
Maybe this should not be here since you seem to use it after calling this function
https://github.com/ubiquibot/conversation-rewards/pull/109/files#diff-f67f7f3e3b4bebb90033a016828f0d24ef83245ba4b26fdb7a83aa1a09466834R61

Reply:
you're right. I just pushed a fix to remove that and also fixed some mock results that I've messed up.